code/script.js

```diff
@@ -1,213 +1,222 @@
 // All the DOM selectors stored as short variables
```

Comment:
Impressive js-file! So clean feels proffesional and with very good comments. 
I really like the greeting alert and that the player's name is displayed while playing. 

In summary, your code is brilliant!

Reply:
Thank you very much Mamite for your kind words!